src/js/components/Pagination/stories/Custom.js

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+
+import { Box, Grommet, Text } from 'grommet';
+import { grommet } from 'grommet/themes';
+import { deepMerge } from 'grommet/utils';
+import { FormPreviousLink, FormNextLink } from 'grommet-icons';
+import { ThemeContext } from 'styled-components';
+import { hpe } from 'grommet-theme-hpe';
+import { Pagination } from '../Pagination';
+
+const customTheme = deepMerge(grommet, {
+  pagination: {
+    button: {
+      color: 'text-strong',
+      active: {
+        background: {
+          color: 'salmon',
+        },
+        color: 'text',
+      },
+      hover: {
+        background: {
+          color: 'skyblue',
+        },
+      },
+    },
+    controls: {
+      gap: 'xxsmall',
+    },
+    icons: {
+      next: FormNextLink,
+      previous: FormPreviousLink,
+    },
+  },
+});
+
+const secondaryTheme = deepMerge(hpe, {
+  pagination: {
+    button: 'secondary',
+  },
+});
+
+const primaryTheme = deepMerge(hpe, {
+  pagination: {
+    button: 'primary',
+  },
+});
```

Comment:
This is nice for the POC, but we should potentially remove it from the grommet's storybook.

Reply:
Agreed, using this for demo purposes but will adjust so it's not hpe specific.

Reply:
Done.